source/infrastructure/monitoring/statuscake/index.html.md.erb

```diff
@@ -30,50 +30,96 @@ It is possible to have a number of emails and mobile numbers for alerts when a s
 It is possible for a Slack message to be raised. You will need a Slack Webhook, which can be provided on request by [Digital tools](<%= data.site.digital_tools %>).
 
 ### Delivery
-It is recommended to use Terraform:
+It is recommended to use Terraform. Example configuration below.
 
 *tfvars* file
 
 ```hcl
-alerts = { # Several different checks per environment can be configured
-  GiT_App_Production_Healthcheck = {
-    website_name  = "Get Into Teaching Website (Production)"                    # Name of the service and environment
-    website_url   = "https://getintoteaching.education.gov.uk/healthcheck.json" # Simple healtheck page with no dependency
-    test_type     = "HTTP"
-    check_rate    = 30                                                          # Set to minimum
-    contact_group = ["Get-Into-Teaching"]
-    trigger_rate  = 0                                                           # Set to 0 to alert immediately
-    confirmations = 1                                                           # Set to 1 or more to eliminate false alarms
-    node_locations = [ "UKINT", "UK1", "MAN1", "MAN5", "DUB2"],                 # Limit the number of check locations
-  }
+"alerts": {                                     # Several different checks per environment can be configured
+  "publish": {
+    "website_name": "publish-teacher-training-prod",                                    # Name of the service and environment
+    "website_url": "https://www.publish-teacher-training-courses.service.gov.uk/ping",  # Simple healtheck page with no dependency
+    "check_rate": 60,
+    "contact_group":  [555000]
+  },
 }
 ```
 
+*tf* configuration
+
 ```hcl
+
 provider "statuscake" {
-  username = var.sc-username
-  apikey   = var.sc-password
+  api_token = local.infra_secrets.STATUSCAKE_PASSWORD
 }
 
 terraform {
-  required_version = ">= 0.13.4"
+  required_version = "1.2.3"
   required_providers {
     statuscake = {
       source  = "StatusCakeDev/statuscake"
-      version = "1.0.1"
+      version = "2.0.3"
     }
   }
 
-resource "statuscake_test" "alert" {
+resource "statuscake_uptime_check" "alert" {
```

Comment:
This code is actually not complete without the `variables` block. Can we add a basic one?

Reply:
Added variables block